src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkDirectional.java

```diff
@@ -333,7 +333,7 @@ public static ItemStack getDirectionalSlotPane(@Nonnull BlockFace blockFace, @No
 
     @Nonnull
     public static ItemStack getDirectionalSlotPane(@Nonnull BlockFace blockFace, @Nonnull Material blockMaterial, boolean active) {
-        if (blockMaterial.isItem() && !blockMaterial.isAir()) {
+        if (blockMaterial.isItem() && !blockMaterial.isAir() && blockMaterial != Material.BEDROCK)) {
```

Comment:
I would say lets use Hardness, that way the same would be true for End Portal Frames etc.

```suggestion
        if (blockMaterial.isItem() && !blockMaterial.isAir() && blockMaterial.getHardness() >= 0)) {
```